shell/app-shell/elements/arc-planner.js

```diff
@@ -63,9 +63,10 @@ class ArcPlanner extends Xen.Debug(Xen.Base, log) {
   }
   async _createPlanificator(arc, userid) {
     const planificatorParam = ArcsUtils.getUrlParam('planificator');
+    const onlyConsumer = ArcsUtils.getUrlParam('onlyConsumer') || false;
```

Comment:
maybe switch to 
 ArcsUtils.getUrlParam('onlyConsumer') === 'true' to force a boolean value?

Reply:
done